### ISSUE TITLE
Exception in REPL

### ISSUE THREAD

**Issue by huguesv** [Microsoft/PTVS#157]
VS 2015RC
PTVS 2.2 30515.00

I opened REPL for Python 2.7.
Typed:
```
$
help
help()
```

That gave me the interactive help, and an activity error message box popped up.  I can't seem to repro it again.

From the activity log:
```
  <entry>
    <record>541</record>
    <time>2015/05/15 16:17:40.725</time>
    <type>Error</type>
    <source>Editor or Editor Extension</source>
    <description>
System.ArgumentOutOfRangeException: Index was out of range. Must be non-negative and less than the size of the collection. Parameter name: index 
at System.ThrowHelper.ThrowArgumentOutOfRangeException(ExceptionArgument argument, ExceptionResource resource) 
at Microsoft.PythonTools.Repl.ReplOutputClassifier.GetClassificationSpans(SnapshotSpan span) 
at Microsoft.VisualStudio.Text.Classification.Implementation.ClassifierTagger.<GetTags>d__1.MoveNext() 
at Microsoft.VisualStudio.Text.Tagging.Implementation.TagAggregator`1.<GetTagsForBuffer>d__1.MoveNext()
    </description>
  </entry>

```

**Comment by huguesv** [comment]
I now have about 25 of these in my activity log.  VS doesn't warn about all of them, even though I've restarted VS several times today.  So it shouldn't be too hard to repro.

**Comment by zooba** [comment]
Should have been fixed by #193
